Worksheet_C/READ.ME.md

```diff
@@ -0,0 +1,9 @@
+The algorithm checks both list I and J and then will check whether I is equal to J and if list [i] is equal to [j]. If they are the same, the list contains a duplicate and will return as true. If there isn’t a duplicate, it will return as being false.
```

Comment:
i and j are not lists, they are indexes into the same list. However you are correct that the algorithm checks if the list contains a duplicate.